hardware_interface/include/hardware_interface/types/hardware_interface_type_values.hpp

```diff
@@ -19,12 +19,20 @@ namespace hardware_interface
 {
 /// Constant defining position interface name
 constexpr char HW_IF_POSITION[] = "position";
+/// Constant defining absolute position interface name
+constexpr char HW_IF_ABSOLUTE_POSITION[] = "absolute_position";
 /// Constant defining velocity interface name
 constexpr char HW_IF_VELOCITY[] = "velocity";
 /// Constant defining acceleration interface name
 constexpr char HW_IF_ACCELERATION[] = "acceleration";
 /// Constant defining effort interface name
 constexpr char HW_IF_EFFORT[] = "effort";
+/// Constant defining torque interface name
+constexpr char HW_IF_TORQUE[] = "torque";
+/// Constant defining force interface name
+constexpr char HW_IF_FORCE[] = "force";
+/// Constant defining current interface name
+constexpr char HW_IF_CURRENT[] = "current";
```

Comment:
Thanks a lot.
This is a great addition.
I would suggest to add temperature, and gains for position and velocity control at low level.

Reply:
I can add temperature. Shouldn't be a problem. What do you mean by gains? like PIDs?

Reply:
Yes in our control boards, the low level control can be done with a feedback on position and speed between 10 KHz and 40 KHz.

Reply:
Such gains can be the results of an algorithm able to compute Ricatti gains.

Reply:
I've just added the temperature part, I'm not sure about what nomenclature we want. 


constexpr char HW_IF_PROPORTIONAL_GAIN[] = "proportional";
constexpr char HW_IF_DERIVATIVE_GAIN[] = "derivative";
constexpr char HW_IF_INTEGRAL_GAIN[] = "integral";
constexpr char HW_IF_FEED_FORWARD[] = "feed_forward";

Reply:
@olivier-stasse Can you propose a nomenclature?

Reply:
I implemented something similar [here](https://github.com/stack-of-tasks/ros2_hardware_interface_odri/blob/4c2f7d8a694782fa34c14f796054add73acb97e4/include/ros2_hardware_interface_odri/system_interface_odri.hpp#L47) but your nomenclature is more informative.

Reply:
Sure

Reply:
@olivier-stasse Do you want me to add the above gain interfaces in this PR? or a new one?

Reply:
I am not sure we need `HW_IF_FEED_FORWARD` because I would assume that a command for a HW_IF_TORQUE interface is the feedforward term.

Reply:
Yes sure, but sometimes you might need to feed_forward acceleration or some entities while doing torque control etc. If you think it doesn't make sense, I can remove it.

Regarding my question, do you want me to include the gain interfaces in this PR or a new one?

Reply:
It would be very nice to have the gain interfaces in this PR.

Reply:
Thanks a lot for porting this @saikishor (with all the other PRs).

Reply:
Hello @olivier-stasse!

I've just added them as per your request :+1: